src/modules/emails/emails.service.test.ts

```diff
@@ -0,0 +1,42 @@
+import { PostEmailsGenerator } from '@ukef-test/support/generator/post-emails-generator';
+import { RandomValueGenerator } from '@ukef-test/support/generator/random-value-generator';
+import { resetAllWhenMocks, when } from 'jest-when';
+
+import { GovukNotifyService } from '../../helper-modules/govuk-notify/govuk-notify.service';
+import { EmailsService } from './emails.service';
+
+jest.mock('@ukef/modules/informatica/informatica.service');
```

Comment:
Do we need this?

Reply:
Good catch, we don't need `jest.mock('@ukef/modules/informatica/informatica.service');`